### ISSUE TITLE
Track progress feature

### ISSUE THREAD

**Issue by jakoss** [armutyus/ninova#25]
How about tracking progress of currently read books? I'd imagine this as marking some book as "currently reading" and then marking the progress by providing percentage (for ebooks/audiobooks) or current page + page count (for physical books).

**Comment by armutyus** [comment]
Thanks for your feedback. Actually this is a cool feature I'll try to add in future releases.

**Comment by jakoss** [comment]
Another useful thing would be to use open library in parallel to google books api. I found both lacking, but together they are pretty solid

**Comment by armutyus** [comment]
Yes, some things are missing on Google API. I even noticed a few more APIs but maybe I will use it when making Discover page.

**Comment by jakoss** [comment]
I did some project revolving the books, and open library was definitely the best in terms of quantity, they had almost anything
